<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>CaseControlAF Vignette ‚Ä¢ CaseControlAF</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="CaseControlAF Vignette"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">CaseControlAF</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/CaseControlAF.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/wolffha/CaseControlAF/" class="external-link">
    <span class="fab fa-github fa-lg"></span>

  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->



      </header>





<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>CaseControlAF Vignette</h1>
                        <h4 data-toc-skip class="author">Hayley
Stoneman</h4>
            
            <h4 data-toc-skip class="date">2024-07-03</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/wolffha/CaseControlAF/blob/HEAD/vignettes/CaseControlAF.Rmd" class="external-link"><code>vignettes/CaseControlAF.Rmd</code></a></small>
      <div class="hidden name"><code>CaseControlAF.Rmd</code></div>

    </div>

    
    
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p><em>Motivation</em></p>
<p>Growth in the field of genetic and genomic research has vastly
increased the amount of available data which is often made publicly
available through summary statistics. While there are many opportunities
and new methods to use summary-level genetic data, it is often limited
by the statistics that are made available. Many post-hoc analyses of
disease data require case and control allele frequencies (AFs), which
are not always published. We present a framework to derive the case and
control AFs from Genome Wide Association Study (GWAS) summary statistics
using the whole sample (case and control aggregated) AF, odds ratio, and
case and control sample sizes, and compare it to a previously published
method that uses the standard error (SE), odds ratio, and case and
control sample sizes to recapitulate case and control AFs.</p>
<p><em>Results</em></p>
<p>In simulations and real data, we find estimating case and control AFs
using the whole sample AF is highly accurate across all settings. When
applying this method to the Pan-UK Biobank we find high concordance with
the known AFs. Conversely, we find that in Pan-UK Biobank and
simulations including covariates, deriving case and control AFs from the
SE underestimates the minor AF (MAF) for higher MAFs. To enable
estimation of case and control AFs using SE, we use gnomAD v3.1.2 AFs as
a proxy for true AFs to estimate and correct for bias fit. While
estimating the case control AF using the whole sample AF is preferred
due to its high accuracy, estimating from the SE can be used more
broadly since the SE can be calculated from the p-value and beta
estimate, which are more commonly provided. Both methods expand the
utility of publicly available genetic summary statistics, can be used
when different statistics are reported, and promote the reusability of
genomic data.</p>
<p><br><br> This document introduces you to the functions in
CaseControlAF and gives small examples of how they can be used with
summary statistics.</p>
<p><br><br> Find more details and documentation on GitHub: (<a
href="https://wolffha.github.io/CaseControlAF/"
class="uri">https://wolffha.github.io/CaseControlAF/</a>)</p>
<p><br><br> To jump forward to individual function specifics:</p>
<p><a href="#CaesControl_AF"><strong>CaseControl_AF</strong></a> ‚Äì <a
href="#a-quick-demo-of-CaseControl_AF">fast forward to example</a> <a
href="#CaesControl_SE"><strong>CaseControl_SE</strong></a> ‚Äì <a
href="#a-quick-demo-of-CaseControl_SE">fast forward to example</a>
<br><br><br></p>
</div>
<div id="installation" class="section level1">
<h1>Installation</h1>
<p>To install this package from BioConductor:</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">require</span>(<span class="st">&quot;BiocManager&quot;</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>))</span>
<span id="cb1-2"><a href="#cb1-2" tabindex="-1"></a>    <span class="fu">install.packages</span>(<span class="st">&quot;BiocManager&quot;</span>)</span>
<span id="cb1-3"><a href="#cb1-3" tabindex="-1"></a></span>
<span id="cb1-4"><a href="#cb1-4" tabindex="-1"></a>BiocManager<span class="sc">::</span><span class="fu">install</span>(<span class="st">&quot;CaseControlAF&quot;</span>)</span></code></pre></div>
<p>To install this package from GitHub:</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" tabindex="-1"></a><span class="cf">if</span>(<span class="sc">!</span><span class="fu">require</span>(devtools, <span class="at">quietly =</span> <span class="cn">TRUE</span>)) {</span>
<span id="cb2-2"><a href="#cb2-2" tabindex="-1"></a>  <span class="fu">install.packages</span>(<span class="st">&quot;devtools&quot;</span>)</span>
<span id="cb2-3"><a href="#cb2-3" tabindex="-1"></a>}</span>
<span id="cb2-4"><a href="#cb2-4" tabindex="-1"></a></span>
<span id="cb2-5"><a href="#cb2-5" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">install_github</span>(<span class="st">&quot;https://github.com/wolffha/CaseControlAF&quot;</span>)</span></code></pre></div>
<div id="overview-of-casecontrolaf-functions" class="section level2">
<h2>Overview of CaseControlAF Functions</h2>
<p>CaseControlAF contains two functions:</p>
<ul>
<li><strong>CaseControl_AF()</strong></li>
<li><strong>CaseControl_SE()</strong></li>
</ul>
<p>The two functions are both used to reconstruct case and control
allele frequencies (AFs) from genetic summary statistics.</p>
<p>The user will select which function to use based on the available
summary statistics.</p>
<p><strong>CaseControl_AF()</strong> has the following required
parameters:</p>
<ul>
<li>Number of cases</li>
<li>Number of controls</li>
<li>Odds Ratio (OR) or beta coefficient</li>
<li><strong>AF</strong> for the whole sample (cases and controls
combined)</li>
</ul>
<p><strong>CaseControl_SE()</strong> has the follwoing required
parameters:</p>
<ul>
<li>Number of cases</li>
<li>Number of controls</li>
<li>Odds Ratio (OR) or beta coefficient</li>
<li><strong>SE</strong> of the log(OR) for each variant</li>
</ul>
<p><em>Code adapted from ReACt GroupFreq function available here: (<a
href="https://github.com/Paschou-Lab/ReAct/blob/main/GrpPRS_src/CountConstruct.c"
class="uri">https://github.com/Paschou-Lab/ReAct/blob/main/GrpPRS_src/CountConstruct.c</a>)</em></p>
</div>
</div>
<div id="CaseControl_AF" class="section level1">
<h1>CaseControl_AF</h1>
<div id="casecontrol_af-input" class="section level2">
<h2>CaseControl_AF() input</h2>
<ul>
<li><strong>data</strong>: a dataframe with each row being a variant and
columns for total AF and OR</li>
<li><strong>N_case</strong>: an integer for the number of case
samples</li>
<li><strong>N_control</strong>: an integer for the number of control
samples</li>
<li><strong>OR_colname</strong>: a string containing the exact column
name in <em>data</em> with the odds ratios</li>
<li><strong>AF_total_colname</strong>: a string containing the exact
column name in <em>data</em> with the total AFs</li>
</ul>
</div>
<div id="casecontrol_af-output" class="section level2">
<h2>CaseControl_AF() output</h2>
<p>Returns the original <em>data</em> dataframe with two additional
columns:</p>
<ul>
<li>AF_case</li>
<li>AF_control</li>
</ul>
<p>The number of rows is equal to the number of variants.</p>
</div>
</div>
<div id="CaseControl_SE" class="section level1">
<h1>CaseControl_SE</h1>
<div id="casecontrol_se-input" class="section level2">
<h2>CaseControl_SE() input</h2>
<p>CaseControl_SE has the following required inputs:</p>
<ul>
<li><strong>data</strong>: a dataframe where each row is a variant and
columns for the OR, SE, chromosome, and position</li>
<li><strong>N_case</strong>: an integer for the number of case
samples</li>
<li><strong>N_control</strong>: an integer for the number of control
samples</li>
<li><strong>OR_colname</strong>: a string containing the exact column
name in <em>data</em> with the odds ratios</li>
<li><strong>SE_colname</strong>: a string containing the exact column
name in <em>data</em> with the standard errors</li>
<li><strong>position_colname</strong>: a string containing the exact
column name in <em>data</em> with the positions of the variants</li>
<li><strong>chromosome_colname</strong>: a string containing the exact
column name in <em>data</em> with the chromosome of the variants. Note,
sex chromosomes can be either characters (‚ÄòX‚Äô, ‚Äòx‚Äô, ‚ÄòY‚Äô, ‚Äòy‚Äô) or numeric
where X=23 and Y=24</li>
<li><strong>sex_chromosomes</strong>: boolean, TRUE if variants from sex
chromosome(s) are included in the dataset</li>
<li><strong>do_correction</strong>: boolean, TRUE if data is provided to
correct the estimates using proxy MAFs</li>
<li><strong>remove_sex_chromosomes</strong>: boolean, TRUE if variants
on sex chromosomes should be removed. This is only necessary if
<em>sex_chromosomes</em> == TRUE and the number of XX/XY individuals per
case and control sample is NOT known</li>
</ul>
<p>CaseControl_SE has the following optional inputs:</p>
<p>If <em>sex_chromosomes</em> == TRUE and
<em>remove_sex_chromosomes</em> == FALSE, then the following inputs are
required:</p>
<ul>
<li><strong>N_XX_case</strong>: the number of XX chromosome case
individuals</li>
<li><strong>N_XX_control</strong>: the number of XX chromosome control
individuals</li>
<li><strong>N_XY_case</strong>: the number of XY chromosome case
individuals</li>
<li><strong>N_XY_control</strong>: the number of XY chromosome control
individuals</li>
</ul>
<p>If <em>do_correction</em> == TRUE, then data must be provided that
includes harmonized data with proxy MAFs</p>
<ul>
<li><strong>correction_data</strong>: a dataframe with the following
EXACT column names: CHR, POS, proxy_MAF, containing data for variants
harmonized between the observed and proxy datasets</li>
</ul>
</div>
<div id="casecontrol_se-output" class="section level2">
<h2>CaseControl_SE() output</h2>
<p>Returns the <em>data</em> dataframe with three additional
columns:</p>
<ul>
<li>MAF_case</li>
<li>MAF_control</li>
<li>MAF_total</li>
</ul>
<p>Each columns contains the estimated MAF in the cases, controls, and
whole sample, respectively. The number of rows is equal to the number of
variants</p>
<p>If <em>do_correction</em> == TRUE, three additional columns are
included in the dataframe:</p>
<ul>
<li>MAF_case_adj</li>
<li>MAF_control_adj</li>
<li>MAF_total_adj</li>
</ul>
<p>These are the MAFs adjusted using the framework described here using
the proxy MAFs to model the expected bias.</p>
<p><em>NOTE:</em> This method assumes we are estimating the minor allele
frequency (MAF). The minor allele may or may not be the effect variant
used to calculate AF for the GWAS summary statistics. For additional
details see vignette titled ‚ÄúCaseControlAF Extra Details‚Äù</p>
</div>
</div>
<div id="examples" class="section level1">
<h1>Examples</h1>
<p>These examples use the provided sample data which is a subset of 500
variants from chromosome 1 of the Pan-UKBB diabetes GWAS in non-Finnish
European individuals.</p>
<div id="a-quick-demo-of-CaseControl_AF" class="section level2">
<h2>A quick demo of CaseControl_AF()</h2>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" tabindex="-1"></a><span class="fu">library</span>(CaseControlAF)</span>
<span id="cb3-2"><a href="#cb3-2" tabindex="-1"></a></span>
<span id="cb3-3"><a href="#cb3-3" tabindex="-1"></a><span class="co"># load the data</span></span>
<span id="cb3-4"><a href="#cb3-4" tabindex="-1"></a><span class="fu">data</span>(<span class="st">&quot;sampleDat&quot;</span>)</span>
<span id="cb3-5"><a href="#cb3-5" tabindex="-1"></a>sampleDat <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(sampleDat)</span>
<span id="cb3-6"><a href="#cb3-6" tabindex="-1"></a></span>
<span id="cb3-7"><a href="#cb3-7" tabindex="-1"></a>results_af <span class="ot">&lt;-</span> <span class="fu">CaseControl_AF</span>(<span class="at">data =</span> sampleDat,</span>
<span id="cb3-8"><a href="#cb3-8" tabindex="-1"></a>                             <span class="at">N_case =</span> <span class="dv">16550</span>,</span>
<span id="cb3-9"><a href="#cb3-9" tabindex="-1"></a>                             <span class="at">N_control =</span> <span class="dv">403923</span>,</span>
<span id="cb3-10"><a href="#cb3-10" tabindex="-1"></a>                             <span class="at">OR_colname =</span> <span class="st">&quot;OR&quot;</span>,</span>
<span id="cb3-11"><a href="#cb3-11" tabindex="-1"></a>                             <span class="at">AF_total_colname =</span> <span class="st">&quot;true_maf_pop&quot;</span>)</span>
<span id="cb3-12"><a href="#cb3-12" tabindex="-1"></a></span>
<span id="cb3-13"><a href="#cb3-13" tabindex="-1"></a><span class="fu">head</span>(results_af)</span>
<span id="cb3-14"><a href="#cb3-14" tabindex="-1"></a><span class="co">#&gt;    CHR       POS REF ALT true_maf_case true_maf_control       beta      SE</span></span>
<span id="cb3-15"><a href="#cb3-15" tabindex="-1"></a><span class="co">#&gt; 1 chr1 226824710   C   T     2.048e-01        2.041e-01  0.0003441 0.01471</span></span>
<span id="cb3-16"><a href="#cb3-16" tabindex="-1"></a><span class="co">#&gt; 2 chr1 117812346   G   A     4.480e-01        4.471e-01 -0.0013920 0.01196</span></span>
<span id="cb3-17"><a href="#cb3-17" tabindex="-1"></a><span class="co">#&gt; 3 chr1 230838863   C   T     1.647e-01        1.642e-01  0.0042250 0.01620</span></span>
<span id="cb3-18"><a href="#cb3-18" tabindex="-1"></a><span class="co">#&gt; 4 chr1  93121792 ATT   A     0.000e+00        0.000e+00 -0.1058000 3.53000</span></span>
<span id="cb3-19"><a href="#cb3-19" tabindex="-1"></a><span class="co">#&gt; 5 chr1 240236388   G   A     1.114e-05        3.775e-05 -1.2540000 1.25300</span></span>
<span id="cb3-20"><a href="#cb3-20" tabindex="-1"></a><span class="co">#&gt; 6 chr1  12385196   G   A     1.390e-02        1.402e-02 -0.0152700 0.05046</span></span>
<span id="cb3-21"><a href="#cb3-21" tabindex="-1"></a><span class="co">#&gt;    gnomad_maf        OR true_maf_pop      AF_case   AF_control</span></span>
<span id="cb3-22"><a href="#cb3-22" tabindex="-1"></a><span class="co">#&gt; 1 2.04050e-01 1.0003442 2.041276e-01 2.041813e-01 2.041254e-01</span></span>
<span id="cb3-23"><a href="#cb3-23" tabindex="-1"></a><span class="co">#&gt; 2 4.45508e-01 0.9986090 4.471354e-01 4.468049e-01 4.471490e-01</span></span>
<span id="cb3-24"><a href="#cb3-24" tabindex="-1"></a><span class="co">#&gt; 3 1.82612e-01 1.0042339 1.642197e-01 1.647775e-01 1.641968e-01</span></span>
<span id="cb3-25"><a href="#cb3-25" tabindex="-1"></a><span class="co">#&gt; 4 1.07925e-04 0.8996046 0.000000e+00 0.000000e+00 0.000000e+00</span></span>
<span id="cb3-26"><a href="#cb3-26" tabindex="-1"></a><span class="co">#&gt; 5 8.82145e-05 0.2853611 3.670262e-05 1.077692e-05 3.776488e-05</span></span>
<span id="cb3-27"><a href="#cb3-27" tabindex="-1"></a><span class="co">#&gt; 6 1.15995e-02 0.9848460 1.401528e-02 1.381395e-02 1.402353e-02</span></span></code></pre></div>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb4-2"><a href="#cb4-2" tabindex="-1"></a><span class="co">#&gt; ‚îÄ‚îÄ [1mAttaching core tidyverse packages[22m ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ tidyverse 2.0.0 ‚îÄ‚îÄ</span></span>
<span id="cb4-3"><a href="#cb4-3" tabindex="-1"></a><span class="co">#&gt; [32m‚úî[39m [34mdplyr    [39m 1.1.4     [32m‚úî[39m [34mreadr    [39m 2.1.5</span></span>
<span id="cb4-4"><a href="#cb4-4" tabindex="-1"></a><span class="co">#&gt; [32m‚úî[39m [34mforcats  [39m 1.0.0     [32m‚úî[39m [34mstringr  [39m 1.5.1</span></span>
<span id="cb4-5"><a href="#cb4-5" tabindex="-1"></a><span class="co">#&gt; [32m‚úî[39m [34mggplot2  [39m 3.5.1     [32m‚úî[39m [34mtibble   [39m 3.2.1</span></span>
<span id="cb4-6"><a href="#cb4-6" tabindex="-1"></a><span class="co">#&gt; [32m‚úî[39m [34mlubridate[39m 1.9.3     [32m‚úî[39m [34mtidyr    [39m 1.3.1</span></span>
<span id="cb4-7"><a href="#cb4-7" tabindex="-1"></a><span class="co">#&gt; [32m‚úî[39m [34mpurrr    [39m 1.0.2     </span></span>
<span id="cb4-8"><a href="#cb4-8" tabindex="-1"></a><span class="co">#&gt; ‚îÄ‚îÄ [1mConflicts[22m ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ tidyverse_conflicts() ‚îÄ‚îÄ</span></span>
<span id="cb4-9"><a href="#cb4-9" tabindex="-1"></a><span class="co">#&gt; [31m‚úñ[39m [34mdplyr[39m::[32mfilter()[39m masks [34mstats[39m::filter()</span></span>
<span id="cb4-10"><a href="#cb4-10" tabindex="-1"></a><span class="co">#&gt; [31m‚úñ[39m [34mdplyr[39m::[32mlag()[39m    masks [34mstats[39m::lag()</span></span>
<span id="cb4-11"><a href="#cb4-11" tabindex="-1"></a><span class="co">#&gt; [36m‚Ñπ[39m Use the conflicted package ([3m[34m&lt;http://conflicted.r-lib.org/&gt;[39m[23m) to force all conflicts to become errors</span></span>
<span id="cb4-12"><a href="#cb4-12" tabindex="-1"></a><span class="co"># plot the results</span></span>
<span id="cb4-13"><a href="#cb4-13" tabindex="-1"></a><span class="co"># first need minor allele frequency to compare to true_maf_case</span></span>
<span id="cb4-14"><a href="#cb4-14" tabindex="-1"></a>results_af<span class="sc">$</span>MAF_case <span class="ot">&lt;-</span> <span class="fu">sapply</span>(results_af<span class="sc">$</span>AF_case, <span class="cf">function</span>(x)</span>
<span id="cb4-15"><a href="#cb4-15" tabindex="-1"></a>  <span class="fu">ifelse</span>(x <span class="sc">&gt;</span> <span class="fl">0.5</span>, <span class="dv">1</span><span class="sc">-</span>x, x))</span>
<span id="cb4-16"><a href="#cb4-16" tabindex="-1"></a>results_af <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(results_af)</span>
<span id="cb4-17"><a href="#cb4-17" tabindex="-1"></a></span>
<span id="cb4-18"><a href="#cb4-18" tabindex="-1"></a><span class="fu">ggplot</span>(results_af, <span class="fu">aes</span>(<span class="at">x =</span> true_maf_case, <span class="at">y =</span> MAF_case)) <span class="sc">+</span></span>
<span id="cb4-19"><a href="#cb4-19" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb4-20"><a href="#cb4-20" tabindex="-1"></a>  <span class="fu">geom_abline</span>(<span class="at">slope =</span> <span class="dv">1</span>, <span class="at">intercept =</span> <span class="dv">0</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">color =</span> <span class="st">&quot;red&quot;</span>) <span class="sc">+</span></span>
<span id="cb4-21"><a href="#cb4-21" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb4-22"><a href="#cb4-22" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">&quot;Example CaseControl_AF()&quot;</span>)</span></code></pre></div>
<p><img src="C:/Users/HayBa/Documents/R%20Packages/CaseControlAF/vignettes/man/figures/unnamed-chunk-4-1.png" width="100%" /></p>
</div>
<div id="a-quick-demo-of-CaseControl_SE" class="section level2">
<h2>A quick demo of CaseControl_SE()</h2>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" tabindex="-1"></a><span class="fu">library</span>(CaseControlAF)</span>
<span id="cb5-2"><a href="#cb5-2" tabindex="-1"></a></span>
<span id="cb5-3"><a href="#cb5-3" tabindex="-1"></a><span class="co"># load the data</span></span>
<span id="cb5-4"><a href="#cb5-4" tabindex="-1"></a><span class="fu">data</span>(<span class="st">&quot;sampleDat&quot;</span>)</span>
<span id="cb5-5"><a href="#cb5-5" tabindex="-1"></a>sampleDat <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(sampleDat)</span>
<span id="cb5-6"><a href="#cb5-6" tabindex="-1"></a></span>
<span id="cb5-7"><a href="#cb5-7" tabindex="-1"></a><span class="co"># First run without correction</span></span>
<span id="cb5-8"><a href="#cb5-8" tabindex="-1"></a>results_se_noCorr <span class="ot">&lt;-</span> <span class="fu">CaseControl_SE</span>(<span class="at">data =</span> sampleDat,</span>
<span id="cb5-9"><a href="#cb5-9" tabindex="-1"></a>                                    <span class="at">N_case =</span> <span class="dv">16550</span>,</span>
<span id="cb5-10"><a href="#cb5-10" tabindex="-1"></a>                                    <span class="at">N_control =</span> <span class="dv">403923</span>,</span>
<span id="cb5-11"><a href="#cb5-11" tabindex="-1"></a>                                    <span class="at">OR_colname =</span> <span class="st">&quot;OR&quot;</span>,</span>
<span id="cb5-12"><a href="#cb5-12" tabindex="-1"></a>                                    <span class="at">SE_colname =</span> <span class="st">&quot;SE&quot;</span>,</span>
<span id="cb5-13"><a href="#cb5-13" tabindex="-1"></a>                                    <span class="at">chromosome_colname =</span> <span class="st">&quot;CHR&quot;</span>,</span>
<span id="cb5-14"><a href="#cb5-14" tabindex="-1"></a>                                    <span class="at">position_colname =</span> <span class="st">&quot;POS&quot;</span>,</span>
<span id="cb5-15"><a href="#cb5-15" tabindex="-1"></a>                                    <span class="at">do_correction =</span> <span class="cn">FALSE</span>,</span>
<span id="cb5-16"><a href="#cb5-16" tabindex="-1"></a>                                    <span class="at">sex_chromosomes =</span> <span class="cn">FALSE</span>)</span>
<span id="cb5-17"><a href="#cb5-17" tabindex="-1"></a><span class="fu">head</span>(results_se_noCorr)</span>
<span id="cb5-18"><a href="#cb5-18" tabindex="-1"></a><span class="co">#&gt;    CHR       POS REF ALT true_maf_case true_maf_control       beta      SE</span></span>
<span id="cb5-19"><a href="#cb5-19" tabindex="-1"></a><span class="co">#&gt; 1 chr1 226824710   C   T     2.048e-01        2.041e-01  0.0003441 0.01471</span></span>
<span id="cb5-20"><a href="#cb5-20" tabindex="-1"></a><span class="co">#&gt; 2 chr1 117812346   G   A     4.480e-01        4.471e-01 -0.0013920 0.01196</span></span>
<span id="cb5-21"><a href="#cb5-21" tabindex="-1"></a><span class="co">#&gt; 3 chr1 230838863   C   T     1.647e-01        1.642e-01  0.0042250 0.01620</span></span>
<span id="cb5-22"><a href="#cb5-22" tabindex="-1"></a><span class="co">#&gt; 4 chr1  93121792 ATT   A     0.000e+00        0.000e+00 -0.1058000 3.53000</span></span>
<span id="cb5-23"><a href="#cb5-23" tabindex="-1"></a><span class="co">#&gt; 5 chr1 240236388   G   A     1.114e-05        3.775e-05 -1.2540000 1.25300</span></span>
<span id="cb5-24"><a href="#cb5-24" tabindex="-1"></a><span class="co">#&gt; 6 chr1  12385196   G   A     1.390e-02        1.402e-02 -0.0152700 0.05046</span></span>
<span id="cb5-25"><a href="#cb5-25" tabindex="-1"></a><span class="co">#&gt;    gnomad_maf        OR true_maf_pop     MAF_case  MAF_control    MAF_total</span></span>
<span id="cb5-26"><a href="#cb5-26" tabindex="-1"></a><span class="co">#&gt; 1 2.04050e-01 1.0003442 2.041276e-01 1.764905e-01 1.764405e-01 1.764425e-01</span></span>
<span id="cb5-27"><a href="#cb5-27" tabindex="-1"></a><span class="co">#&gt; 2 4.45508e-01 0.9986090 4.471354e-01 3.263835e-01 3.266896e-01 3.266776e-01</span></span>
<span id="cb5-28"><a href="#cb5-28" tabindex="-1"></a><span class="co">#&gt; 3 1.82612e-01 1.0042339 1.642197e-01 1.392353e-01 1.387297e-01 1.387496e-01</span></span>
<span id="cb5-29"><a href="#cb5-29" tabindex="-1"></a><span class="co">#&gt; 4 1.07925e-04 0.8996046 0.000000e+00 2.513875e-06 2.794421e-06 2.783379e-06</span></span>
<span id="cb5-30"><a href="#cb5-30" tabindex="-1"></a><span class="co">#&gt; 5 8.82145e-05 0.2853611 3.670262e-05 1.946826e-05 6.821993e-05 6.630105e-05</span></span>
<span id="cb5-31"><a href="#cb5-31" tabindex="-1"></a><span class="co">#&gt; 6 1.15995e-02 0.9848460 1.401528e-02 1.250050e-02 1.269041e-02 1.268294e-02</span></span></code></pre></div>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb6-2"><a href="#cb6-2" tabindex="-1"></a><span class="co"># plot the results</span></span>
<span id="cb6-3"><a href="#cb6-3" tabindex="-1"></a><span class="fu">ggplot</span>(results_se_noCorr, <span class="fu">aes</span>(<span class="at">x =</span> true_maf_case, <span class="at">y =</span> MAF_case)) <span class="sc">+</span></span>
<span id="cb6-4"><a href="#cb6-4" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb6-5"><a href="#cb6-5" tabindex="-1"></a>  <span class="fu">geom_abline</span>(<span class="at">slope =</span> <span class="dv">1</span>, <span class="at">intercept =</span> <span class="dv">0</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">color =</span> <span class="st">&quot;red&quot;</span>) <span class="sc">+</span></span>
<span id="cb6-6"><a href="#cb6-6" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb6-7"><a href="#cb6-7" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">&quot;Example CaseControl_SE() No Correction&quot;</span>)</span></code></pre></div>
<p><img src="C:/Users/HayBa/Documents/R%20Packages/CaseControlAF/vignettes/man/figures/unnamed-chunk-5-1.png" width="100%" /></p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" tabindex="-1"></a><span class="fu">library</span>(CaseControlAF)</span>
<span id="cb7-2"><a href="#cb7-2" tabindex="-1"></a></span>
<span id="cb7-3"><a href="#cb7-3" tabindex="-1"></a><span class="co"># load the data</span></span>
<span id="cb7-4"><a href="#cb7-4" tabindex="-1"></a><span class="fu">data</span>(<span class="st">&quot;sampleDat&quot;</span>)</span>
<span id="cb7-5"><a href="#cb7-5" tabindex="-1"></a>sampleDat <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(sampleDat)</span>
<span id="cb7-6"><a href="#cb7-6" tabindex="-1"></a></span>
<span id="cb7-7"><a href="#cb7-7" tabindex="-1"></a>corr_data <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">CHR =</span> sampleDat<span class="sc">$</span>CHR,</span>
<span id="cb7-8"><a href="#cb7-8" tabindex="-1"></a>                        <span class="at">POS =</span> sampleDat<span class="sc">$</span>POS,</span>
<span id="cb7-9"><a href="#cb7-9" tabindex="-1"></a>                        <span class="at">proxy_MAF =</span> sampleDat<span class="sc">$</span>gnomad_maf)</span>
<span id="cb7-10"><a href="#cb7-10" tabindex="-1"></a></span>
<span id="cb7-11"><a href="#cb7-11" tabindex="-1"></a><span class="co"># now run with correction</span></span>
<span id="cb7-12"><a href="#cb7-12" tabindex="-1"></a>results_se_corr <span class="ot">&lt;-</span> <span class="fu">CaseControl_SE</span>(<span class="at">data =</span> sampleDat,</span>
<span id="cb7-13"><a href="#cb7-13" tabindex="-1"></a>                                  <span class="at">N_case =</span> <span class="dv">16550</span>,</span>
<span id="cb7-14"><a href="#cb7-14" tabindex="-1"></a>                                  <span class="at">N_control =</span> <span class="dv">403923</span>,</span>
<span id="cb7-15"><a href="#cb7-15" tabindex="-1"></a>                                  <span class="at">OR_colname =</span> <span class="st">&quot;OR&quot;</span>,</span>
<span id="cb7-16"><a href="#cb7-16" tabindex="-1"></a>                                  <span class="at">SE_colname =</span> <span class="st">&quot;SE&quot;</span>,</span>
<span id="cb7-17"><a href="#cb7-17" tabindex="-1"></a>                                  <span class="at">chromosome_colname =</span> <span class="st">&quot;CHR&quot;</span>,</span>
<span id="cb7-18"><a href="#cb7-18" tabindex="-1"></a>                                  <span class="at">position_colname =</span> <span class="st">&quot;POS&quot;</span>,</span>
<span id="cb7-19"><a href="#cb7-19" tabindex="-1"></a>                                  <span class="at">do_correction =</span> <span class="cn">TRUE</span>,</span>
<span id="cb7-20"><a href="#cb7-20" tabindex="-1"></a>                                  <span class="at">correction_data =</span> corr_data,</span>
<span id="cb7-21"><a href="#cb7-21" tabindex="-1"></a>                                  <span class="at">sex_chromosomes =</span> <span class="cn">FALSE</span>)</span>
<span id="cb7-22"><a href="#cb7-22" tabindex="-1"></a></span>
<span id="cb7-23"><a href="#cb7-23" tabindex="-1"></a><span class="fu">head</span>(results_se_corr)</span>
<span id="cb7-24"><a href="#cb7-24" tabindex="-1"></a><span class="co">#&gt;    CHR       POS REF ALT true_maf_case true_maf_control       beta      SE</span></span>
<span id="cb7-25"><a href="#cb7-25" tabindex="-1"></a><span class="co">#&gt; 1 chr1 226824710   C   T     2.048e-01        2.041e-01  0.0003441 0.01471</span></span>
<span id="cb7-26"><a href="#cb7-26" tabindex="-1"></a><span class="co">#&gt; 2 chr1 117812346   G   A     4.480e-01        4.471e-01 -0.0013920 0.01196</span></span>
<span id="cb7-27"><a href="#cb7-27" tabindex="-1"></a><span class="co">#&gt; 3 chr1 230838863   C   T     1.647e-01        1.642e-01  0.0042250 0.01620</span></span>
<span id="cb7-28"><a href="#cb7-28" tabindex="-1"></a><span class="co">#&gt; 4 chr1  93121792 ATT   A     0.000e+00        0.000e+00 -0.1058000 3.53000</span></span>
<span id="cb7-29"><a href="#cb7-29" tabindex="-1"></a><span class="co">#&gt; 5 chr1 240236388   G   A     1.114e-05        3.775e-05 -1.2540000 1.25300</span></span>
<span id="cb7-30"><a href="#cb7-30" tabindex="-1"></a><span class="co">#&gt; 6 chr1  12385196   G   A     1.390e-02        1.402e-02 -0.0152700 0.05046</span></span>
<span id="cb7-31"><a href="#cb7-31" tabindex="-1"></a><span class="co">#&gt;    gnomad_maf        OR true_maf_pop     MAF_case  MAF_control    MAF_total</span></span>
<span id="cb7-32"><a href="#cb7-32" tabindex="-1"></a><span class="co">#&gt; 1 2.04050e-01 1.0003442 2.041276e-01 1.764905e-01 1.764405e-01 1.764425e-01</span></span>
<span id="cb7-33"><a href="#cb7-33" tabindex="-1"></a><span class="co">#&gt; 2 4.45508e-01 0.9986090 4.471354e-01 3.263835e-01 3.266896e-01 3.266776e-01</span></span>
<span id="cb7-34"><a href="#cb7-34" tabindex="-1"></a><span class="co">#&gt; 3 1.82612e-01 1.0042339 1.642197e-01 1.392353e-01 1.387297e-01 1.387496e-01</span></span>
<span id="cb7-35"><a href="#cb7-35" tabindex="-1"></a><span class="co">#&gt; 4 1.07925e-04 0.8996046 0.000000e+00 2.513875e-06 2.794421e-06 2.783379e-06</span></span>
<span id="cb7-36"><a href="#cb7-36" tabindex="-1"></a><span class="co">#&gt; 5 8.82145e-05 0.2853611 3.670262e-05 1.946826e-05 6.821993e-05 6.630105e-05</span></span>
<span id="cb7-37"><a href="#cb7-37" tabindex="-1"></a><span class="co">#&gt; 6 1.15995e-02 0.9848460 1.401528e-02 1.250050e-02 1.269041e-02 1.268294e-02</span></span>
<span id="cb7-38"><a href="#cb7-38" tabindex="-1"></a><span class="co">#&gt;   MAF_case_adj MAF_control_adj MAF_total_adj</span></span>
<span id="cb7-39"><a href="#cb7-39" tabindex="-1"></a><span class="co">#&gt; 1 2.101173e-01    2.100673e-01  2.100692e-01</span></span>
<span id="cb7-40"><a href="#cb7-40" tabindex="-1"></a><span class="co">#&gt; 2 4.831277e-01    4.834338e-01  4.834218e-01</span></span>
<span id="cb7-41"><a href="#cb7-41" tabindex="-1"></a><span class="co">#&gt; 3 1.598885e-01    1.593830e-01  1.594029e-01</span></span>
<span id="cb7-42"><a href="#cb7-42" tabindex="-1"></a><span class="co">#&gt; 4 2.513875e-06    2.794421e-06  2.783379e-06</span></span>
<span id="cb7-43"><a href="#cb7-43" tabindex="-1"></a><span class="co">#&gt; 5 1.946826e-05    6.821993e-05  6.630105e-05</span></span>
<span id="cb7-44"><a href="#cb7-44" tabindex="-1"></a><span class="co">#&gt; 6 8.977955e-03    9.167862e-03  9.160387e-03</span></span></code></pre></div>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb8-2"><a href="#cb8-2" tabindex="-1"></a><span class="co"># plot the results</span></span>
<span id="cb8-3"><a href="#cb8-3" tabindex="-1"></a><span class="fu">ggplot</span>(results_se_corr, <span class="fu">aes</span>(<span class="at">x =</span> true_maf_case, <span class="at">y =</span> MAF_case_adj)) <span class="sc">+</span></span>
<span id="cb8-4"><a href="#cb8-4" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb8-5"><a href="#cb8-5" tabindex="-1"></a>  <span class="fu">geom_abline</span>(<span class="at">slope =</span> <span class="dv">1</span>, <span class="at">intercept =</span> <span class="dv">0</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">color =</span> <span class="st">&quot;red&quot;</span>) <span class="sc">+</span></span>
<span id="cb8-6"><a href="#cb8-6" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb8-7"><a href="#cb8-7" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">&quot;Example CaseControl_SE() With Correction&quot;</span>)</span></code></pre></div>
<p><img src="C:/Users/HayBa/Documents/R%20Packages/CaseControlAF/vignettes/man/figures/unnamed-chunk-6-1.png" width="100%" /></p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span>
<span id="cb9-2"><a href="#cb9-2" tabindex="-1"></a><span class="co">#&gt; R version 4.4.1 (2024-06-14 ucrt)</span></span>
<span id="cb9-3"><a href="#cb9-3" tabindex="-1"></a><span class="co">#&gt; Platform: x86_64-w64-mingw32/x64</span></span>
<span id="cb9-4"><a href="#cb9-4" tabindex="-1"></a><span class="co">#&gt; Running under: Windows 11 x64 (build 22631)</span></span>
<span id="cb9-5"><a href="#cb9-5" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb9-6"><a href="#cb9-6" tabindex="-1"></a><span class="co">#&gt; Matrix products: default</span></span>
<span id="cb9-7"><a href="#cb9-7" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb9-8"><a href="#cb9-8" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb9-9"><a href="#cb9-9" tabindex="-1"></a><span class="co">#&gt; locale:</span></span>
<span id="cb9-10"><a href="#cb9-10" tabindex="-1"></a><span class="co">#&gt; [1] LC_COLLATE=English_United States.utf8 </span></span>
<span id="cb9-11"><a href="#cb9-11" tabindex="-1"></a><span class="co">#&gt; [2] LC_CTYPE=English_United States.utf8   </span></span>
<span id="cb9-12"><a href="#cb9-12" tabindex="-1"></a><span class="co">#&gt; [3] LC_MONETARY=English_United States.utf8</span></span>
<span id="cb9-13"><a href="#cb9-13" tabindex="-1"></a><span class="co">#&gt; [4] LC_NUMERIC=C                          </span></span>
<span id="cb9-14"><a href="#cb9-14" tabindex="-1"></a><span class="co">#&gt; [5] LC_TIME=English_United States.utf8    </span></span>
<span id="cb9-15"><a href="#cb9-15" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb9-16"><a href="#cb9-16" tabindex="-1"></a><span class="co">#&gt; time zone: America/Denver</span></span>
<span id="cb9-17"><a href="#cb9-17" tabindex="-1"></a><span class="co">#&gt; tzcode source: internal</span></span>
<span id="cb9-18"><a href="#cb9-18" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb9-19"><a href="#cb9-19" tabindex="-1"></a><span class="co">#&gt; attached base packages:</span></span>
<span id="cb9-20"><a href="#cb9-20" tabindex="-1"></a><span class="co">#&gt; [1] stats     graphics  grDevices utils     datasets  methods   base     </span></span>
<span id="cb9-21"><a href="#cb9-21" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb9-22"><a href="#cb9-22" tabindex="-1"></a><span class="co">#&gt; other attached packages:</span></span>
<span id="cb9-23"><a href="#cb9-23" tabindex="-1"></a><span class="co">#&gt;  [1] lubridate_1.9.3      forcats_1.0.0        stringr_1.5.1       </span></span>
<span id="cb9-24"><a href="#cb9-24" tabindex="-1"></a><span class="co">#&gt;  [4] dplyr_1.1.4          purrr_1.0.2          readr_2.1.5         </span></span>
<span id="cb9-25"><a href="#cb9-25" tabindex="-1"></a><span class="co">#&gt;  [7] tidyr_1.3.1          tibble_3.2.1         ggplot2_3.5.1       </span></span>
<span id="cb9-26"><a href="#cb9-26" tabindex="-1"></a><span class="co">#&gt; [10] tidyverse_2.0.0      CaseControlAF_0.99.0</span></span>
<span id="cb9-27"><a href="#cb9-27" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb9-28"><a href="#cb9-28" tabindex="-1"></a><span class="co">#&gt; loaded via a namespace (and not attached):</span></span>
<span id="cb9-29"><a href="#cb9-29" tabindex="-1"></a><span class="co">#&gt;  [1] sass_0.4.9        utf8_1.2.4        generics_0.1.3    stringi_1.8.4    </span></span>
<span id="cb9-30"><a href="#cb9-30" tabindex="-1"></a><span class="co">#&gt;  [5] hms_1.1.3         digest_0.6.36     magrittr_2.0.3    timechange_0.3.0 </span></span>
<span id="cb9-31"><a href="#cb9-31" tabindex="-1"></a><span class="co">#&gt;  [9] evaluate_0.24.0   grid_4.4.1        fastmap_1.2.0     jsonlite_1.8.8   </span></span>
<span id="cb9-32"><a href="#cb9-32" tabindex="-1"></a><span class="co">#&gt; [13] fansi_1.0.6       scales_1.3.0      textshaping_0.4.0 jquerylib_0.1.4  </span></span>
<span id="cb9-33"><a href="#cb9-33" tabindex="-1"></a><span class="co">#&gt; [17] cli_3.6.3         rlang_1.1.4       munsell_0.5.1     withr_3.0.1      </span></span>
<span id="cb9-34"><a href="#cb9-34" tabindex="-1"></a><span class="co">#&gt; [21] cachem_1.1.0      yaml_2.3.10       tools_4.4.1       tzdb_0.4.0       </span></span>
<span id="cb9-35"><a href="#cb9-35" tabindex="-1"></a><span class="co">#&gt; [25] colorspace_2.1-1  vctrs_0.6.5       R6_2.5.1          lifecycle_1.0.4  </span></span>
<span id="cb9-36"><a href="#cb9-36" tabindex="-1"></a><span class="co">#&gt; [29] fs_1.6.4          htmlwidgets_1.6.4 ragg_1.3.2        pkgconfig_2.0.3  </span></span>
<span id="cb9-37"><a href="#cb9-37" tabindex="-1"></a><span class="co">#&gt; [33] desc_1.4.3        pkgdown_2.1.0     pillar_1.9.0      bslib_0.8.0      </span></span>
<span id="cb9-38"><a href="#cb9-38" tabindex="-1"></a><span class="co">#&gt; [37] gtable_0.3.5      glue_1.7.0        systemfonts_1.1.0 highr_0.11       </span></span>
<span id="cb9-39"><a href="#cb9-39" tabindex="-1"></a><span class="co">#&gt; [41] xfun_0.45         tidyselect_1.2.1  rstudioapi_0.16.0 knitr_1.48       </span></span>
<span id="cb9-40"><a href="#cb9-40" tabindex="-1"></a><span class="co">#&gt; [45] farver_2.1.2      htmltools_0.5.8.1 labeling_0.4.3    rmarkdown_2.27   </span></span>
<span id="cb9-41"><a href="#cb9-41" tabindex="-1"></a><span class="co">#&gt; [49] compiler_4.4.1</span></span></code></pre></div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p><p>Developed by Hayley Wolff.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.0.</p>
</div>

      </footer></div>






  </body></html>
